assets/src/components/header.tsx

```diff
@@ -37,6 +39,23 @@ export const Header = () => {
           </ReactLink>
         );
       })}
+      <button
+        onClick={
+          () => {
+            if (!window.confirm('Are you sure you want to logout?')) {
+              return;
+            }
+
+            useLogoutMutation.mutateAsync(undefined, undefined).then(
+              () => {
+                window.location.reload();
+              },
+            );
+          }
+        }
```

Comment:
separate to `handleLogout`

Reply:
done 1981c11fabf1b21a92ead789c950bf23ce067a6a